### DISCUSSION TITLE
Get a Galaxy Brain badge !!

### DISCUSSION THREAD

**Discussion by mayacx** [community/community#38187] [Other Feature Feedback, Questions, & Ideas]
How do I get the Galaxy Brain badge?

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by ProgrammerVrushabh** [comment]
Just answer the questions as many as possible on discussion section of github if any questions get marked as answer you will get your badge .
By the way you can mark as answer this 😊

Thanks !
